migrate.sh

```diff
@@ -0,0 +1,12 @@
+export OLD_VERSION="v1.128.0"
+export NEW_VERSION="v1.129.0"
+wget "https://github.com/immich-app/immich/releases/download/$NEW_VERSION/docker-compose.yml"
+wget "https://github.com/immich-app/immich/releases/download/$NEW_VERSION/example.env"
+wget "https://github.com/immich-app/immich/releases/download/$NEW_VERSION/hwaccel.ml.yml"
+wget "https://github.com/immich-app/immich/releases/download/$NEW_VERSION/hwaccel.transcoding.yml"
+wget "https://github.com/immich-app/immich/releases/download/$NEW_VERSION/prometheus.yml"
```

Comment:
why download all the files while we only use `docker-compose.yml` ?

Also, please cleanup the files afterwards or make a temporary file